fundbot/plugins/fund/fund.py

```diff
@@ -15,10 +15,15 @@ async def handle_fund(bot: Bot, event: Event, state: T_State):
     args = str(event.get_message()).split()
     result = ""
     if len(args) == 0:
-        await fund.finish("用法: #fund <fundid>")
```

Comment:
I suggest that separate these two usage, and make them two command, such as `fund` and `fund_hist`.